packages/api/src/modules/users/router.ts

```diff
@@ -133,4 +144,21 @@ router.post(
     }
 );
 
+router.post(
+    "/updatedescription",
```

Comment:
```suggestion
    "/updateDescription",
```

camelCase please.

Reply:
well its a route doesnt matter thatt much but I fixed it anyway 😄